.circleci/scripts/remove_stacks.py

```diff
@@ -0,0 +1,28 @@
+import boto3
+import datetime
+import os
+
+from dateutil.tz import tzutc
+
+
+def check_conditions(stack: dict, target_stacks: dict, time_diff: datetime.timedelta):
```

Comment:
non-blocking: could we rename this function to `check_stack_timeout`. Also could we update the function args to describe the types a bit more:

```
def check_stack_timeout(stack: Dict[str, Any], target_stacks: Dict[str, List[Any]], timeout: datetime.timedelta) -> None:
```